app/views/api/services/forms/_authentication_settings.html.slim

```diff
@@ -0,0 +1,40 @@
+= f.inputs class: 'auth-settings-wrapper' do
+  - # oidc
```

Comment:
I assume that it is not your fault but all these 'ruby comments' should be slim comments, so for example instead of `- # oidc`, it should be `/ oidc` and then it is not rendered either.